plugins/adr/src/components/EntityAdrContent/EntityAdrContent.tsx

```diff
@@ -95,29 +193,20 @@ export const EntityAdrContent = (props: {
       return;
     }
 
-    const adrs = value.data
-      .filter(
-        (item: { type: string; name: string }) =>
-          item.type === 'file' &&
-          (filePathFilterFn
-            ? filePathFilterFn(item.name)
-            : madrFilePathFilter(item.name)),
-      )
-      .map(({ name, title, status, date }) => ({ name, title, status, date }));
+    const adrs: AdrFileInfo[] = value.data.filter(
+      (item: AdrFileInfo) =>
+        item.type === 'file' &&
+        (filePathFilterFn
+          ? filePathFilterFn(item.name)
+          : madrFilePathFilter(item.name)),
```

Comment:
We probably want to update here to use `path` instead. It also occurred to me that we will want update the default [madrFilePathFilter](https://github.com/backstage/backstage/blob/master/plugins/adr-common/src/index.ts#L79-L80) method to account for  the directory prefix for this to work correctly with paths